README.md

```diff
@@ -236,21 +236,20 @@ you can do so with a simple environment variable, instead of editing the
         -R, --require path               Additional file to require before loading models, may be used multiple times
         -e [tests,fixtures,factories,serializers],
             --exclude                    Do not annotate fixtures, test files, factories, and/or serializers
-        -f [bare|rdoc|yard|markdown],    Render Schema Infomation as plain/RDoc/YARD/Markdown
+        -f [bare|rdoc|yard|markdown],    Render Schema Infomation as plain/RDoc/Yard/Markdown
             --format
             --force                      Force new annotations even if there are no changes.
             --frozen                     Do not allow to change annotations. Exits non-zero if there are going to be changes to files.
             --timestamp                  Include timestamp in (routes) annotation
             --trace                      If unable to annotate a file, print the full stack trace, not just the exception message.
-        -I, --ignore-columns REGEX       don't annotate columns that match a given REGEX (e.g. `annotate -I '^(id|updated_at|created_at)'`)
-            --ignore-routes REGEX        don't annotate routes that match a given REGEX (e.g. `annotate -I '(mobile|resque|pghero)'`)_
+        -I, --ignore-columns REGEX       don't annotate columns that match a given REGEX (i.e., `annotate -I '^(id|updated_at|created_at)'`
```

Comment:
The original is correct here; these are examples. 

* i.e. (latin: _id est_), means "in other words" 
* e.g. (latin: _exemplī grātiā_), means "for example"

If we substitute the meaning for the abbreviation, it becomes clearer which one fits.

> ... don't annotate columns that match a given REGEX (in other words, `annotate -I '^...`

> ... don't annotate columns that match a given REGEX (for example, `annotate -I '^...`